subsys/bluetooth/host/smp.c

```diff
@@ -307,8 +307,7 @@ static bool le_sc_supported(void)
 		return false;
 	}
 
-	return BT_CMD_TEST(bt_dev.supported_commands, 34, 1) &&
-	       BT_CMD_TEST(bt_dev.supported_commands, 34, 2);
+	return IS_ENABLED(CONFIG_BT_ECC);
```

Comment:
The comment above here is now out of date. It can probably be removed.

Also, can this function be simplified now that `BT_ECC` is promtless and selected if `!BT_SMP_OOB_LEGACY_PAIR_ONLY`?

Reply:
Yes, most likely this entire function and the related state variable can be removed.

Reply:
Actually, I'll leave the function for now, but with the incorrect comment removed, 
We could e.g. have Mesh as an enabler of `BT_ECC` in the future but still have `BT_SMP_OOB_LEGACY_PAIR_ONLY` enabled, in which case we wouldn't want SC to only dependent on `BT_ECC`.